### PR TITLE
Fix repeated word

### PR DESCRIPTION
**Pull request by dcorderoch**

### CODE REVIEW

**Comment by entorb** [comment]
Thanks! Just checked for more via `[^’]\b([a-z]+)\b \1\b` and found these, what do you think about them?

`Hint hint` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-063.tex#L719

`that that` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-088.tex#L444

`at at` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-104.tex#L413

**Comment by dcorderoch** [comment]
> Thanks! Just checked for more via `[^’]\b([a-z]+)\b \1\b` and found these, what do you think about them?
> 
> `Hint hint` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-063.tex#L719
> 
> `that that` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-088.tex#L444
> 
> `at at` at https://github.com/rrthomas/hpmor/blob/main/chapters/hpmor-chapter-104.tex#L413

missed those, I'll add them too!

File: chapters/hpmor-chapter-063.tex [hunk 1]
```diff
@@ -716,7 +716,7 @@ \section{Aftermath, Hermione Granger}
 
 And if the Defence Professor \emph{did} have a flaw in his sanity, it was that his outlook on life was \emph{too negative}.
 
-\emph{Really?} inquired the part of Harry that had read eighteen million experimental results about people being too optimistic and overconfident. \emph{Professor Quirrell is too pessimistic? So pessimistic that his expectations routinely \emph{undershoot} reality? Stuff him and put him in a museum, he’s unique. Which one of you two planned the perfect crime, and \emph{then} put in all the error margin and fallbacks that ended up saving your butt, \emph{just in case} the perfect crime went wrong? Hint hint, his name wasn’t Harry Potter.}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-063.tex:719]: This one is correct, please revert.

Comment (dcorderoch) [reply chapters/hpmor-chapter-063.tex]: done!

**Pull request merged**